widget/penrose/euclidean.dsl

```diff
@@ -0,0 +1,6 @@
+type Targettable
+type Point <: Targettable
+type Line <: Targettable
```

Comment:
It doesn't look like you are using `Targettable` in the style. If so, it can just be deleted.
```suggestion
type Point
type Line
```

Reply:
It seems that `Targettable` styling is actually prepended to the style program in `penroseCanvas.tsx`, so the visualisation does not work after removing it from the dsl. I reverted this back to the original version.

Reply:
This is something I had been meaning to fix. It is no longer needed now.